modes/simple-mpc/evil-collection-simple-mpc.el

```diff
@@ -0,0 +1,79 @@
+;;; evil-collection-mpc.el --- Bindings for `simple-mpc-mode' -*- lexical-binding: t -*-
+;; Copyright (C) 2022 Joseph Turner
+
+;; Author: Joseph Turner <joseph@breatheoutbreathe.in>
+;; Maintainer: Joseph Turner <joseph@breatheoutbreathe.in>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "27.2"))
+;; Keywords: evil, simple-mpc, processes, mpd
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;;; Bindings for `simple-mpc-mode'.
+
+;;; Code:
+(require 'simple-mpc)
+(require 'evil-collection)
+
+(defconst evil-collection-simple-mpc-mode-maps '(simple-mpc-mode-map))
+
+;;;###autoload
+(defun evil-collection-simple-mpc-setup ()
+  "Setup up 'evil' bindings for 'simple-mpc-mode', 'simple-mpc-query-mode', and 'simple-mpc-current-playlist-mode'."
+  (evil-collection-define-key 'normal 'simple-mpc-mode-map
+    "p" 'simple-mpc-toggle
+    ">" 'simple-mpc-next
+    "<" 'simple-mpc-prev
+    "f" 'simple-mpc-seek-forward
+    "b" 'simple-mpc-seek-backward
+    "s" 'simple-mpc-query
+    "c" 'simple-mpc-view-current-playlist
+    "l" 'simple-mpc-load-playlist
+    "z" 'simple-mpc-shuffle-current-playlist
```

Comment:
`z` is a prefix key. Maybe `S` is a good candidate.

Reply:
`S` is fine too. I only chose `z` because that's what ncmpcpp uses for their random mode.